src/test/ui/const-generics/generic_const_exprs/issue-72787.rs

```diff
@@ -7,10 +7,11 @@ pub struct IsLessOrEqual<const LHS: u32, const RHS: u32>;
 pub struct Condition<const CONDITION: bool>;
 pub trait True {}
 
-impl<const LHS: u32, const RHS: u32> True for IsLessOrEqual<LHS, RHS> where
-    Condition<{ LHS <= RHS }>: True
+impl<const LHS: u32, const RHS: u32>
+True for IsLessOrEqual<LHS, RHS> where Condition<{ LHS <= RHS }>: True
```

Comment:
Was this formatting change intentional? Seems like it'd be best to use rustfmt's formatting here.